lib/sort_by_length.rb

```diff
@@ -1,7 +1,20 @@
 # A method which will return an array of the words in the string
 #  sorted by the length of the word.
-# Time complexity: ?
-# Space complexity: ?
+# Time complexity: O(n^2)
+# Space complexity: O(n)
 def sort_by_length(my_sentence)
```

Comment:
See my comment below, but I want to reiterate that you should look closely at my example sorting methods.  Pay attention to swapping elements in the array.  That can reduce your time complexity a lot.